bin/_mocha

```diff
@@ -66,10 +66,10 @@ program
   .option('-c, --colors', 'force enabling of colors')
   .option('-C, --no-colors', 'force disabling of colors')
   .option('-G, --growl', 'enable growl notification support')
-  .option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options')
-  .option('-R, --reporter <name>', 'specify the reporter to use', 'spec')
-  .option('-S, --sort', 'sort test files')
-  .option('-b, --bail', 'bail after first test failure')
+  .option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options, use name1:{k=v,k2=v2,...},name2:{...} for multiple reporters')
```

Comment:
I think I may have mentioned this before, but this format is not something people are accustomed to.  can we please use the [subarg](https://www.npmjs.com/package/subarg) format (for lack of anything better that I know of)?  you'll probably want to pull in that package.